modular_skyrat/modules/conveyor_sorter/code/conveyor_sorter.dm

```diff
@@ -145,3 +146,71 @@
 		"conveysorter",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+
+/obj/item/conveyor_sorter/improved
+	name = "improved conveyor sorter lister"
+	desc = "A tool that is used to not only create the conveyor sorters, but give lists to the conveyor sorters. This one has an increased capacity of eight."
+	icon_state = "lister_improved"
+
+/obj/item/conveyor_sorter/improved/examine(mob/user)
+	. = ..()
+	. += span_notice("This version of the sorter allows up to eight sorters to be placed.")
+
+/obj/item/conveyor_sorter/improved/attack_self(mob/user, modifiers)
+	if(length(spawned_sorters) >= 8)
+		to_chat(user, span_warning("You may only have eight spawned conveyor sorters!"))
```

Comment:
and if it was a var, it would prevent the need for this kind of thing

Reply:
ah i see, yea i can do that real quick

Reply:
same as above response just a momet ago but relevant all the same

Reply:
![image](https://user-images.githubusercontent.com/2568378/215297273-68692c86-9117-4fb3-863e-d617ff6d1abb.png)